db/seeds.rb

```diff
@@ -13,57 +13,42 @@
 USERS_PER_EVENT = 5
 
 NUM_USERS.times do
-  User.create!({
+  User.create!(
     name: Faker::Name.unique.name,
     email: Faker::Internet.unique.email,
     password: Faker::Internet.password
-  })
+  )
 end
 
 NUM_LOCATIONS.times do
-  Location.create!({
+  Location.create!(
     name: Faker::Address.unique.community,
     address: Faker::Address.unique.street_address
-  })
+  )
 end
 
 LOCATIONS = Location.all
 
 NUM_EVENTS.times do
-  event = Event.create!({
+  event = Event.create!(
     name: Faker::Pokemon.unique.name,
     duration_minutes: rand(300),
     location_id: LOCATIONS.sample.id
-  })
+  )
+end
 
-  # indices = []
-  #
-  # (6 * USERS_PER_EVENT).times do
-  #   indices << rand(NUM_USERS - 1)
-  # end
-  #
-  # all_users = User.all
-  # event_users = indices.uniq[0...USERS_PER_EVENT].map { |index| all_users[index] }
-  #
-  # event_users.each do |user|
-  #   EventsUser.create!({
-  #     event_id: event.id,
-  #     user_id: user.id
-  #   })
-  # end
+USERS = User.all
 
+Event.all.each do |event|
+  event.users = USERS.sample(USERS_PER_EVENT)
```

Comment:
Nice!

Reply:
Oh, hound got on me in another PR for using `Model.all.each`. Should we use `Event.find_each` instead?